.gitignore

```diff
@@ -59,6 +59,9 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 
+# E2E Test logs
+geckodriver.log
+
```

Comment:
geckodriver itself should also be ignored - it's a binary file that you don't manage, is OS specific, and devs might have their own geckodriver already.

Reply:
And also pytestdebug.log